### PR TITLE
devops: deprecated h_set_nest_theme

### PR DESCRIPTION
**Pull request by kpagacz**
Related to insightsengineering/coredev-tasks#291

Edit: relies on https://github.com/insightsengineering/nestcolor/pull/12

### CODE REVIEW

File: R/individual_patient_plot.R [hunk 1]
```diff
@@ -36,6 +36,12 @@ NULL
 #' @export
 #'
 h_set_nest_theme <- function(font_size) {
+  lifecycle::deprecate_soft(
+    what = "h_set_nest_theme()",
+    details = "h_set_nest_theme() will be removed in a future release.",
+    with = "nestcolor::theme_nest()",
+    when = "0.7.9"
+  )
```
Comment (nikolas-burkoff) [inline R/individual_patient_plot.R:44]: If `h_set_nest_theme` and `nestcolor::theme_nest()` provide the same functionality as `nestcolor` is in the imports of tern could we replace the rest of the function with a call to `nestcolor::theme_nest()`?

If they provide different functionality then of course we can't

File: inst/WORDLIST [hunk 2]
```diff
@@ -1,14 +1,22 @@
 AE
-ANCOVA
 Agresti
+ANCOVA
 Anova
+biomarker
 Biomarker
-Bové
+biomarkers
+Bov<U+00E9>
```
Comment (nikolas-burkoff) [inline inst/WORDLIST:8]: This has changed - the unicode is probably ok but think é is nicer?

Comment (nikolas-burkoff) [reply inst/WORDLIST:8]: Also further down

**Review by nikolas-burkoff — COMMENTED** [review]
A couple of small comments but then it's good to go

File: inst/WORDLIST (hunk 2 above)
Comment (kpagacz) [reply inst/WORDLIST:8]: It's the `precommit` being a little demon. I need to revert this.

**Review by nikolas-burkoff — APPROVED** [review]

File: R/individual_patient_plot.R (hunk 1 above)
Comment (nikolas-burkoff) [reply R/individual_patient_plot.R:44]: so at the moment `theme_nest` does not have a font size argument so they are not the same thing - I can add one to nest_color and then update this - or I can merge this in now and deal with any other nest_color changes later @shajoezhu

File: NEWS.md [hunk 3]
```diff
@@ -30,98 +30,99 @@
...
    `fct_collapse_only`, `fct_discard`, `fct_explicit_na_if`.
-   
+
 * Deprecated badge added to `g_mmrm`
 * Removed `tern:::` prefix from internal function uses in tests
 
 ### Miscellaneous
-* Deprecated the `color_palette` function with `nestcolor::color_palette` and removed `color_palette_core`
+* Deprecated the `color_palette` function with `nestcolor::color_palette` and removed `color_palette_core`.
+* Deprecated `h_set_nest_theme()` in favour of `nestcolor::theme_nest()`.
```
Comment (nikolas-burkoff) [inline NEWS.md]:
```suggestion
* Deprecated `h_set_nest_theme()` in favor of `nestcolor::theme_nest()`.
```

File: R/individual_patient_plot.R (hunk 1 above)
Comment (shajoezhu) [reply R/individual_patient_plot.R:44]: Hi @nikolas-burkoff , sorry that I missed this message before. Could you please add the font size argument in `theme_nest` please. Thanks so much!

**Review by shajoezhu — APPROVED** [review]
lgtm! Thanks so much @nikolas-burkoff

File: R/individual_patient_plot.R (hunk 1 above)
Comment (nikolas-burkoff) [reply R/individual_patient_plot.R:44]: ok so I have added font_size here https://github.com/insightsengineering/nestcolor/pull/12 and used nestcolor in this function as well now so once the nestcolor PR is approved they both can go in

**Pull request merged**